pkg/server/commands/write_test.go

```diff
@@ -175,6 +203,14 @@ func TestValidateWriteRequest(t *testing.T) {
 					ReadAuthorizationModel(gomock.Any(), gomock.Any(), gomock.Any()).
 					Return(&openfgav1.AuthorizationModel{
 						SchemaVersion: typesystem.SchemaVersion1_1,
+						TypeDefinitions: parser.MustTransformDSLToProto(`
```

Comment:
Can't you just do `testutils.MustTransformDSLToProtoWithID(modelStr)` for short here?